packages/example/src/App.tsx

```diff
@@ -83,63 +91,145 @@ const App = ({ examples, cells, renderers}: AppProps) => {
   const [currentExample, setExample] = useState<ExampleDescription>(examples[0]);
   const [currentIndex, setIndex] = useState<number>(0);
   const [dataAsString, setDataAsString] = useState<any>('');
-  const [props, setProps] = useState<any>(getProps(currentExample, cells, renderers)); // helper function
+  const [schemaAsString, setSchemaAsString] = useState<any>('');
+  const [uiSchemaAsString, setUiSchemaAsString] = useState<any>('');
+  const [props, setProps] = useState<any>(getProps(currentExample, cells, renderers));
+  const [showPanel, setShowPanel] = useState<boolean>(true);
 
-  const actions: any = currentExample.actions;
+  const actions: Action[] = currentExample.actions;
+
+  useEffect(() => {
+    const hash = window.location.hash.replace('#', '');
+    const exampleIndex = examples.findIndex(example => {
+      return example.name === hash
+    });
+    if(exampleIndex !== -1) {
+      changeExample(exampleIndex);
+    }
+  }, []);
 
   const changeExample = (exampleID: number) => {
     let example = examples[exampleID];
     setIndex(exampleID);
     setExample(example);
     setProps(getProps(example, cells, renderers));
+    window.location.hash = example.name;
+    if(example.name == 'huge') {
+      setShowPanel(false);
+    }
   };
 
+  useEffect(() => {
+    if(props.schema != undefined) {
+      setSchemaAsString(JSON.stringify(props.schema, null, 2));
+    } else {
+      setSchemaAsString('No schema set');
+    }
+    if(props.uischema != undefined) {
+      setUiSchemaAsString(JSON.stringify(props.uischema, null, 2));
+    } else {
+      setUiSchemaAsString('No ui schema set');
+    }
+  }, [props.schema, props.uischema]);
+
+  useEffect(() => {
+    var panel = document.getElementsByClassName('props-panel')[0];
+    if(showPanel) {
+      panel.classList.remove('hide');
+    } else {
+      panel.classList.add('hide');
+    }
+  }, [showPanel]);
+
   const changeData = (data: any) => {
     setDataAsString(JSON.stringify(data, null, 2));
   };
 
+  const togglePanelHeight = () => {
+    var panel = document.getElementsByClassName('panel-wrapper')[0];
+    panel.classList.toggle('full-height');
+  }
```

Comment:
This could also be done by handing over different strings to `className` of the panel instead of direct DOM manipulation, i.e.
```
<div className={`someStaticClass ${fullPanelHeight ? 'full-height' : ''}`>
```
It requires managing an additional state variable but that's fine.